tests/Parser/Visitor/Example/DocBlock/ImportedGenericArgument.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
I know it doesn't really matter in this class but every PHP file should `declare(strict_types=1);` - just to be consistent

Reply:
I left it out because none of the existing test classes declared it. But you are right. For good measure I've added the declaration to all the other ones as well.